poker-app/Home.js

```diff
@@ -1,32 +1,33 @@
 import React, { Component } from 'react';
 import { StyleSheet, Text, View, Button, AsyncStorage, TouchableOpacity } from 'react-native';
 import StatsBox from './components/Statsbox'
+const storage = require("./components/AsyncStorageController.js");
 class HomeScreen extends Component {
     state = {
         gamesArray : []
     }
 
     componentDidMount() {
         this.setState({
-            gamesArray: this.retrieveData()
+            gamesArray: storage.retrieveData
```

Comment:
did you mean to execute storage.retrieveData()?

Reply:
yes i did, thx